go.mod

```diff
@@ -250,6 +251,8 @@ replace (
 	// Required by Cortex https://github.com/cortexproject/cortex/pull/3051.
 	github.com/bradfitz/gomemcache => github.com/themihai/gomemcache v0.0.0-20180902122335-24332e2d58ab
 
+	github.com/thanos-io/objstore => github.com/yeya24/objstore v0.0.0-20220922180308-1fe6830f03a6
```

Comment:
I use my temporary fork for now to bring https://github.com/thanos-io/objstore/pull/28. 
This could help unblock the build as it still uses the old version.